_include/q/sbt-quickstart.sh

```diff
@@ -38,11 +38,19 @@ function mkPackage() {
   echo ${x//./\/}
 }
 
+function extract_parameter() {
+  if [[ "$#" != 1 ]]; then
+    echo "Fatal error: No parameter or too many parameter passed: $@"
+    exit 1;
+  fi
+  awk -F'"' '/'$1' = "/{print $2}' docs/config.toml
+}
+
 defaultProjectName="Flink Project"
 defaultOrganization="org.example"
 defaultVersion="0.1-SNAPSHOT"
-defaultScalaVersion="2.12.7"
-defaultFlinkVersion="1.17.0"
+defaultFlinkVersion="$(extract_parameter 'FlinkStableVersion')"
+defaultScalaVersion="$(extract_parameter 'ScalaVersion')"
```

Comment:
```suggestion
defaultScalaVersion="$(extract_parameter 'ScalaVersion')"
defaultFlinkVersion="$(extract_parameter 'FlinkStableVersion')"
```
let's stick to the order to make the diff more readable.